### PR TITLE
[Xsup 26102] ConvertXmlFileToJson script to py3

### PR DESCRIPTION
**Pull request by yucohen**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/XSUP-26102

## Description
Converted the script from js to python, this works for big xml files as well

### CODE REVIEW

File: Packs/CommonScripts/Scripts/ConvertXmlFileToJson/ConvertXmlFileToJson.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import demistomock as demisto
+from CommonServerPython import *
+import xmltodict
+
+
+def convert_file(entry_id: str, verbose: bool, context_key:str):
```
Comment (bziser) [inline Packs/CommonScripts/Scripts/ConvertXmlFileToJson/ConvertXmlFileToJson.py:6]: return type

File: Packs/CommonScripts/Scripts/ConvertXmlFileToJson/ConvertXmlFileToJson.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import demistomock as demisto
+from CommonServerPython import *
+import xmltodict
+
+
+def convert_file(entry_id: str, verbose: bool, context_key:str | None) ->  None:
+    """
+    Converts file content crom XML to Json
```
Comment (bziser) [inline Packs/CommonScripts/Scripts/ConvertXmlFileToJson/ConvertXmlFileToJson.py:8]:
```suggestion
    Converts file content from XML to Json
```

File: Packs/CommonScripts/Scripts/ConvertXmlFileToJson/ConvertXmlFileToJson_test.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import pytest
+import demistomock as demisto
+def test_convert_file(mocker):
```
Comment (bziser) [inline Packs/CommonScripts/Scripts/ConvertXmlFileToJson/ConvertXmlFileToJson_test.py:3]: please add `given when `

**Review by bziser — COMMENTED** [review]
great work 👏

File: Packs/CommonScripts/Scripts/ConvertXmlFileToJson/ConvertXmlFileToJson.py [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+        return_results(xml_json)
+    if context_key:
+        appendContext(key=context_key, data=xml_json)
+
+
+def main():  # pragma: no cover
+    args = demisto.args()
+    entry_id = args.get("entryID", "")
+    verbose = argToBoolean(args.get("verbose", False))
+    context_key = args.get("contextKey")
```
Comment (bziser) [inline Packs/CommonScripts/Scripts/ConvertXmlFileToJson/ConvertXmlFileToJson.py:27]:
```suggestion
    context_key = args.get("contextKey",'')
```

File: Packs/CommonScripts/Scripts/ConvertXmlFileToJson/ConvertXmlFileToJson.py [hunk 5]
```diff
@@ -0,0 +1,35 @@
+import demistomock as demisto
+from CommonServerPython import *
+import xmltodict
+
+
+def convert_file(entry_id: str, verbose: bool, context_key: str | None) -> None:
```
Comment (bziser) [inline Packs/CommonScripts/Scripts/ConvertXmlFileToJson/ConvertXmlFileToJson.py:6]:
```suggestion
def convert_file(entry_id: str, verbose: bool, context_key: str) -> None:
```

**Review by bziser — APPROVED** [review]

**Comment by yucohen** [comment]
needs force merge as the js file was deleted

**Pull request merged**